traffic_monitor/towrap/towrap.go

```diff
@@ -343,7 +343,9 @@ func (s TrafficOpsSessionThreadsafe) CRConfigRaw(cdn string) ([]byte, error) {
 		b, reqInf, e := ss.GetCRConfig(cdn)
 		err = e
 		data = b
-		remoteAddr = reqInf.RemoteAddr.String()
+		if reqInf.RemoteAddr != nil {
+			remoteAddr = reqInf.RemoteAddr.String()
```

Comment:
Should we log the error here and below? If there's an error, should we try to recover or let it fall through?

Reply:
I don't think that value being `nil` hampers the code after it. I could add an error here, but the only other place that `ReqAddr` (which gets set with the `remoteAddr` value) is being used is in the comparison on line 152 in the same file.